doc/manual/metaprogramming.rst

```diff
@@ -545,7 +545,7 @@ This macro can be used like this:
     julia> @assert 1==1.0
 
     julia> @assert 1==0
-    ERROR: AssertionError: 1 == 0
+    ERROR: Assertion failed: 1 == 0
```

Comment:
This would cause the doctest to fail, since the doctest does not include the above section that defines a new `@assert` macro. So it's instead calling the built-in `@assert` macro. Maybe we could modify the above macro definition to be more consistent and say `throw(AssertionError($(string(ex))))` instead of `error("Assertion failed: ", $(string(ex)))` ?